common/config/data.js

```diff
@@ -9,6 +9,47 @@ export const DONATION_ADDRESSES_MAP = {
 
 export const donationAddressMap = DONATION_ADDRESSES_MAP;
 
+export const errorMsgs = [
```

Comment:
Would prefer to import these one-by-one, instead of grepping later for those used.